mlir/lib/Transforms/lab4/safarov_nurlan/countFunctionCallsPass.cpp

```diff
@@ -0,0 +1,47 @@
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Tools/Plugins/PassPlugin.h"
+
+using namespace mlir;
+
+namespace {
+class FunctionCallCounterPass
+    : public PassWrapper<FunctionCallCounterPass, OperationPass<ModuleOp>> {
+public:
+  StringRef getArgument() const final { return "SafarovNcntFuncCalls"; }
+  void runOnOperation() override {
+    std::vector<LLVM::LLVMFuncOp> functions;
+    std::map<StringRef, int> numberOfCalls;
+
+    getOperation()->walk([&](Operation *opearation) {
```

Comment:
```suggestion
    getOperation()->walk([&](Operation *operation) {
```

Reply:
@safarov-nm Could you please fix this typo before we merge it into main branch?

Reply:
Okay.

Reply:
Fixed it. Thanks!